pbft/pbft.cpp

```diff
@@ -392,7 +398,7 @@ pbft::handle_commit(const pbft_msg& msg, const bzn_envelope& original_msg)
 }
 
 void
-pbft::handle_join_or_leave(const pbft_membership_msg& msg)
+pbft::handle_join_or_leave(const pbft_membership_msg& msg, std::shared_ptr<bzn::session_base> session)
 {
     if (!this->is_primary())
     {
```

Comment:
was this copied from handle_request()? requests should be forwarded to the primary now